pkgs/development/python-modules/black-macchiato/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pytestCheckHook,
+  black
+}:
+
+buildPythonPackage rec {
+  pname = "black-macchiato";
+  version = "1.3.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner  = "wbolster";
+    repo   = pname;
+    rev    = version;
+    sha256 = "0lc9w50nlbmlzj44krk7kxcia202fhybbnwfh77xixlc7vb4rayl";
+  };
+
+  checkInputs = [ pytestCheckHook black ];
```

Comment:
```suggestion
  propagatedBuildInputs = [ black ];

  checkInputs = [ pytestCheckHook ];
```
```
[nix-shell:/home/jon/.cache/nixpkgs-review/pr-96976-1]$ nix-shell --pure -p "with import ./nixpkgs { }; black-macchiato"

[nix-shell:/home/jon/.cache/nixpkgs-review/pr-96976-1]$ black-macchiato --help
Traceback (most recent call last):
  File "/nix/store/33fap5zfxiawb8kmpyy6harfidkdqmig-python3.8-black-macchiato-1.3.0/bin/.black-macchiato-wrapped", line 6, in <module>
    from macchiato import main
  File "/nix/store/33fap5zfxiawb8kmpyy6harfidkdqmig-python3.8-black-macchiato-1.3.0/lib/python3.8/site-packages/macchiato.py", line 8, in <module>
    import black
ModuleNotFoundError: No module named 'black'
```

Reply:
a `pythonImportsCheck` would have also caught this, do you mind adding one?

Please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
Interesting - I previously did have a `propagatedBuildInputs = [ black ];`, and subsequently verified that the binary worked by running `echo "    if True: print('hi')" | ./result/bin/black-macchiato`, which yielded the expected result.

That line must have been mistakenly removed during one of my changes and I then didn't run the resulting binary. My apologies. Fixing.